src/proxy-ui-api/frontend/src/views/Clients/ServiceClients/AddServiceClientAccessRightsWizard.vue

```diff
@@ -0,0 +1,140 @@
+
+    <v-stepper :alt-labels="true" v-model="step" class="stepper noshadow">
+      <v-stepper-header class="noshadow stepper-header">
+        <v-stepper-step :complete="step > 1" step="1">{{$t('serviceClients.memberGroupStep')}}</v-stepper-step>
+        <v-divider></v-divider>
+        <v-stepper-step :complete="step > 2" step="2">{{$t('serviceClients.servicesStep')}}</v-stepper-step>
+      </v-stepper-header>
+
+      <v-stepper-items class="stepper-content">
+        <v-stepper-content step="1">
+          <MemberOrGroupSelectionStep
+            :id="this.id"
+            :service-clients="serviceClients"
+            v-on:candidate-selection="candidateSelection"
+            v-on:set-step="nextStep"></MemberOrGroupSelectionStep>
+        </v-stepper-content>
+        <v-stepper-content step="2">
+          <ServiceSelectionStep
+            v-if="serviceClientCandidateSelection"
+            :serviceCandidates="serviceCandidates"
+            :serviceClientCandidateSelection="serviceClientCandidateSelection"
+            :id="id"
+            v-on:set-step="previousStep"></ServiceSelectionStep>
+        </v-stepper-content>
+      </v-stepper-items>
+    </v-stepper>
+
+  </div>
+</template>
+
+<script lang="ts">
+  import Vue from 'vue';
+  import SubViewTitle from '@/components/ui/SubViewTitle.vue';
+  import MemberOrGroupSelectionStep from '@/views/Clients/ServiceClients/MemberOrGroupSelectionStep.vue';
+  import ServiceSelectionStep from '@/views/Clients/ServiceClients/ServiceSelectionStep.vue';
+  import LargeButton from '@/components/ui/LargeButton.vue';
+  import {AccessRight, Service, ServiceClient, ServiceDescription} from '@/types';
+  import * as api from '@/util/api';
+  import {ServiceCandidate} from '@/ui-types';
+
+  export default Vue.extend({
+    props: {
+      id: {
+        type: String,
+        required: true,
+      },
+    },
+    components: {
+      SubViewTitle,
+      MemberOrGroupSelectionStep,
+      ServiceSelectionStep,
+      LargeButton,
+    },
+    data() {
+      return {
+        step: 1 as number,
+        serviceClientAccessRights: [] as AccessRight[],
+        serviceCandidates: [] as ServiceCandidate[],
+        serviceClients: [] as ServiceClient[],
+        serviceClientCandidateSelection: undefined as undefined | ServiceClient,
+      };
+    },
+    methods: {
+      candidateSelection(candidate: ServiceClient): void {
+        this.serviceClientCandidateSelection = candidate;
+      },
+      fetchData(): void {
+
+        api.get(`/clients/${this.id}/service-clients`, {})
+          .then( ( response: any ): void => this.serviceClients = response.data )
+          .catch( (error: any) =>
+            this.$store.dispatch('showError', error));
+
+        api
+          .get(`/clients/${this.id}/service-descriptions`)
+          .then( (response: any) => {
+            const serviceDescriptions = response.data as ServiceDescription[];
+            this.serviceCandidates = serviceDescriptions
```

Comment:
Can you add comments about what is happening in this chain?

Reply:
Added comments